src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -136,7 +149,11 @@ extends AbstractMap[Key, Value]
       None
     } else {
       val res = entry.value
```

Comment:
That's not the kind of change you should make in a point release anyway.  You could have something that relies on the existing behavior that would unexpectedly stop working.